AjaxControlToolkit/Scripts/ModalPopup.js

```diff
@@ -173,7 +173,7 @@ Sys.Extended.UI.ModalPopupBehavior.prototype = {
         }
 
         this._backgroundElement = document.createElement('div');
-        this._backgroundElement.dataset.actControlType = "modalPopupBackground";
+        this._backgroundElement.setAttribute( 'actControlType', 'modalPopupBackground');
```

Comment:
This line generates `actControlType` attribute instead of `data-actControlType`.